layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -106,9 +106,14 @@ automatically applied to."
 (defun spacemacs/toggle-maximize-buffer ()
   "Maximize buffer"
   (interactive)
-  (if (and (= 1 (length (window-list)))
-           (assoc ?_ register-alist))
-      (jump-to-register ?_)
+  (if (let ((window-count (length (window-list))))
+        (and
+         (or ( = 1 window-count)
+             (and
+              ( = 2 window-count)
+              (neo-global--window-exists-p)))
```

Comment:
neotree is not always installed so we cannot expect to have this function always defined.